perllib/FixMyStreet/Script/Reports.pm

```diff
@@ -229,6 +230,10 @@ sub send(;$) {
                 if FixMyStreet->test_mode && $sender->can('open311_test_req_used');
         }
 
+        foreach (@methods) {
```

Comment:
Maybe worth a comment on why this is happening so it doesn't get optimised away again in the future.